src/main/java/frc/robot/command/CalibrateSwerveDriveCommand.java

```diff
@@ -0,0 +1,256 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.command;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystem.DriveSubsystem;
+import frc.robot.util.MoPrefs;
+import frc.robot.util.MoUtils;
+import java.util.ArrayList;
+import java.util.function.DoubleSupplier;
+
+public class CalibrateSwerveDriveCommand extends Command {
```

Comment:
At some point, we should probably move this class (and other swerve classes, such as SwerveModule) to MoUtils.
That doesn't need to happen now, though. Maybe offseason?